modules/guAnalyticsAdapter.js

```diff
@@ -4,6 +4,7 @@ import CONSTANTS from '../src/constants.json';
 import adapterManager from '../src/adapterManager.js';
 import * as utils from '../src/utils.js';
 import {ajax} from '../src/ajax.js';
+import { log } from '@guardian/libs/dist/umd/index.min.js';
```

Comment:
https://github.com/guardian/Prebid.js/pull/113/files

Reply:
The reason is that the loader does not support features like optional chaining, which is where the build fails:

```
./node_modules/@guardian/libs/dist/esm/storage.js
Module parse failed: Unexpected token (52:89)
You may need an appropriate loader to handle this file type.
|     get(key) {
|         try {
|             const { value, expires } = JSON.parse(__classPrivateFieldGet(this, _storage)?.getItem(key) ?? '');
|             // is this item has passed its sell-by-date, remove it
|             if (expires && new Date() > new Date(expires)) {
 @ ./node_modules/@guardian/libs/dist/esm/logger.js 12:0-36
 @ ./modules/guAnalyticsAdapter.js
 @ multi ./modules/guAnalyticsAdapter.js
```

Reply:
Actually, this is expected behaviour for @guardian packages, they should be loaded and transpiled by the consuming library: https://github.com/guardian/recommendations/blob/master/npm-packages.md#using-guardian-npm-packages

Reply:
I've fixed it!